jssha/jssha.d.ts

```diff
@@ -1,13 +1,22 @@
 // Type definitions for jsSHA
 // Project: https://github.com/Caligatio/jsSHA
-// Definitions by: David Li <https://github.com/randombk>
```

Comment:
please keep original author name. like this https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/jquery/jquery.d.ts#L3